Vizzuality/half-earth-v3
# Feature/add marine biodiversity layer

## Fishes priority & marine mammals layers
### Description
This PR adds the tile layers for _fishes priority_, _marine mammals priority, rarity and richness_, and _marine vertebrates priority, rarity and richness_.
It also adds a documentation page, under the `docs` folder, that explains the steps to follow in order to add new layers to the project.
Some changes to file names have been made.
### Testing instructions
Check the biodiversity sidebar and toggle  marine layers. Make sure the metadata for the layers is there after clicking the info buttons.
Read the `docs` file `layers-update-flow.md`.
### Feature relevant tickets
https://vizzuality.atlassian.net/browse/HE-20
